tasks/TakeOutGarbage.tex

```diff
@@ -0,0 +1,61 @@
+\section{Take Out the Garbage [House Holder]}
+The robot has to take out the trash, recycling and compost.
+
+% \subsection{Focus}
+% This test focuses on object perception, manipulation, and planning.
+
+\subsection{Main Goal}
+All garbage in the apartment has has been deposited into the correct exterior bin.
+
+\noindent\textbf{Reward:} 450pts (150pts per type of garbage)
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Opening the apartment door (100pts)
+	\item Closing the apartment door (100pts)
+\end{enumerate}
```

Comment:
Threre's something missing here. The test should round up to 1000pts including all bonuses

Reply:
Ah, I read 3.6.3 "The maximum total score per task in Stage I is 1000 points." as, for each task, the scoring of that task is no more than 1000 points. The correct reading is for each task, the maximum attainable score is equal to 1000pts.